torch/_dynamo/symbolic_convert.py

```diff
@@ -866,7 +880,12 @@ def __init__(cls, name, bases, dct) -> None:
         super().__init__(name, bases, dct)
 
         def _missing(opname, *args):
-            unimplemented(f"missing: {opname}")
+            unimplemented_v2(
+                gb_type="Missing bytecode handler",
+                context=opname,
+                explanation=f"Dynamo does not know how to handle the bytecode instruction {opname}",
+                hints=[],
```

Comment:
"file a github issue" ?

Reply:
Hint provided in generic graph break hints PR.